src/Tvheadend.cpp

```diff
@@ -2470,6 +2470,24 @@ void CTvheadend::ParseRecordingAddOrUpdate ( htsmsg_t *msg, bool bAdd )
   if ((str = htsmsg_get_str(msg, "fanartImage")) != NULL)
     rec.SetFanartImage(GetImageURL(str));
 
+  /* Extra checks for HTSP v32+ */
+  if (m_conn->GetProtocol() >= 32) {
+    /* 
+      Due to changes in HTSP v32, try to use subtitle 
+      as the description if the description and summary
+      failed. Clear the subtitle afterwards  to avoid
+      duplicate information being displayed.
+
+      This was done by TVHeadend prior to HTSP v32.
+    */
```

Comment:
This comment block fits best after `if (rec.GetDescription().empty()) {`, not before. Please move the comment.